atest/regexp_prompt.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Default Tags   pybot   jybot
+Resource       resources/common.robot
+Test Teardown  Close All Connections
+
+*** Test Cases ***
+Read Until Prompt With Regexp Prompt
+    Open Connection  localhost  prompt=[$|#]  prompt_is_regexp=yes
+    Login  test  test
+    Write  pwd
+    ${output}=	Read Until Prompt
+    Should Contain	${output}  $
+    Write  export PS1\='\\u\@\\h \\W # '
+    ${output1}=	Read Until Prompt
```

Comment:
Good, but there is a catch. I think Read until prompt will match the # in `export PS1` command, instead of the prompt. So output will be:
```
export PS1='\u@\h \W # '
test@host ~ # 
```
but read until prompt will read only read `export PS1='\u@\h \W #`. After export command need to consume output with `Read` to not match the command itself.

Reply:
No, the `Write` keyword consumes the command text before returning. I also checked the generated log file, the output of `Read Until Prompt` is just:
``` robotframework
${output1} = test@host ~ #
```
`Write Bare` keyword does not consume the command and would also return the `export PS1='\u@\h \W # '`.

Reply:
Oh, of course, you are right! My mistake. Good, it's way simpler that way.